README.md

```diff
@@ -65,6 +65,16 @@ All tests support the same set of arguments :
   * `-c,--check <0/1>` check correctness of results. This can be quite slow on large numbers of GPUs. Default : 1.
   * `-z,--blocking <0/1>` Make NCCL collective blocking, i.e. have CPUs wait and sync after each collective. Default : 0.
   * `-G,--cudagraph <num graph launches>` Capture iterations as a CUDA graph and then replay specified number of times. Default : 0.
+* Saving results
+  * `-F,--results_file` Define path for CSV file to persist nccl-tests result along with CLI arguments. Directory has to exist! Default: "" (Do not save to CSV file).
+
+## CSV format
+Use `-F data.csv` to persist to CSV format, the use tools like [pandas](https://pandas.pydata.org/) to analyse the results.
+```
+import pandas as pd
+df = pd.read_csv("data.csv", header=[0, 1, 2])
+print(df[('out-of-place', ' time', '(us)')]) # NOTE: we are using hierarchical columns
+```
```

Comment:
Thanks for the example, it's a really nice integration.

Can we modify the text to:

## CSV integration with Pandas
CSV files generated with the `-F` option can be used directly in [pandas](https://pandas.pydata.org/). Here is an example printing the time column:
```
[example]
```